src/main/java/core/basesyntax/services/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.services.interfaces.FileReader;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    private static final String EXCEPTION_MESSAGE = "Can't read file ";
+
+    @Override
+    public List<String> getLinesFromFile(String fileName) {
+        String path = fileName;
+        List<String> dataFromFile = new ArrayList<>();
+        File file = new File(path);
+        if (file.exists()) {
```

Comment:
I'm not sure we have to check the file for existing here, while we using try-catch construction and throw IOException, but I think it's not a mistake.

Reply:
I agree